README.md

```diff
@@ -67,9 +67,9 @@ const uppy = new Uppy({ autoProceed: false })
 $ npm install @uppy/core @uppy/dashboard @uppy/tus
 ```
 
-We recommend installing from npm and then using a module bundler such as [Webpack](https://webpack.js.org/), [Browserify](http://browserify.org/) or [Rollup.js](http://rollupjs.org/).
+We recommend installing from npm and then using a module bundler such as [Vite](https://vitejs.dev/), [Parcel](https://parceljs.org/), or [Rollup](https://rollupjs.org).
```

Comment:
Let's remove this from the readme, to avoid having to maintain a list of bundlers.

Reply:
maybe just keep one, and the one we are using ourselves at that point? i do think there is value in recommending people a bundler here, and linking to one

Reply:
We've discussed this during the Uppy call, and we decided that the README was not the place to recommend bundlers, instead we can keep doing that in the docs, so don't have to update 38 files each time we want to update the list. wdyt?

Reply:
👌